src/types/viewport.cpp

```diff
@@ -137,6 +137,17 @@ COORD Viewport::Origin() const noexcept
     return { Left(), Top() };
 }
 
+// Method Description:
+// - For Accessibility, get a coord representing the end of this viewport in inclusive terms.
+// Arguments:
+// - <none>
+// Return Value:
+// - the coordinates of this viewport's end.
+COORD Viewport::EndInclusive() const noexcept
+{
+    return { Left(), BottomExclusive() };

```

Comment:
This is needed to represent an exclusive endpoint in UiaTextRange that includes the last COORD's text in the buffer at (RightInclusive(), BottomInclusive())

So if we had a text range for the entire text buffer, it would be composed of Origin (0,0) and this (0, BottomExclusive()). Since the end is exclusive.